var/spack/repos/builtin/packages/py-gmxapi/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+
+import spack.build_systems.python
+from spack.package import *
+
+
+class PyGmxapi(spack.build_systems.python.PythonPackage):
+    """Python bindings and ensemble workflow management for GROMACS.
+
+    The GROMACS C++ API is affected by its package variants. You can
+    specify a particular GROMACS API by making the dependency variant explicit.
+    E.g. ``spack install gmxapi ^gromacs+mpi~double``
+    """
+
+    homepage = "https://manual.gromacs.org/current/gmxapi/index.html"
+    maintainers("eirrgang", "peterkasson")
+
+    pypi = "gmxapi/gmxapi-0.4.0.tar.gz"
+    version("0.4.0", sha256="7fd58e6a4b1391043379e8ba55555ebeba255c5b394f5df9d676e6a5571d7eba")
+
+    depends_on("gromacs")
+    depends_on("mpi")
+    depends_on("py-mpi4py")
+    depends_on("py-networkx@2.0:", type="run")
+    depends_on("py-numpy@1.7:", type="run")
+    depends_on("py-setuptools@42:", type="build")
+    depends_on("py-packaging", type="run")
+    depends_on("py-pybind11@2.6:", type="build")
+    depends_on("py-pytest", type="test")
```

Comment:
Missing build:
```toml
requires = ["cmake>=3.16", "mpi4py", "pybind11>=2.6", "setuptools>=42.0", "wheel"]
```
and build/run:
```cfg
install_requires =
        mpi4py
        packaging
        pybind11
        networkx>=2.0
        numpy>1.7
```
dependencies.

Reply:
updated

Reply:
Note: I did not include `pybind11` in the "run" dependencies. It was unnecessary, and will be removed in future package metadata. Ref: https://gitlab.com/gromacs/gromacs/-/merge_requests/3525